docs/docs/dev_docs/sandbox/main.md

```diff
@@ -2,51 +2,376 @@
 
+`/bin/bash -c "$(curl -fsSL https://up.aztec.network)"`
 
-FOR INSTRUCTIONS FOR BUILDING A WALLET, WE SHOULD WRITE DOCS [HERE](../wallets/building_a_wallet.md)
+It will download and execute a script invoking docker compose with 2 containers:
 
-ERRORS:
-- Add any error explanations to [errors.md](./common_errors.md) (and break that file into multiple files if it's too unwieldy).
+- Anvil
+- Aztec Sandbox
 
+It will need to create servers on localhost ports 8545 (Anvil) and 8080 (Sandbox) so you will need to ensure nothing conflicts with this.
 
-- RPC calls directly
-  - Lower priority, but eventually we should answer all of the aztec.js questions here as well.
\ No newline at end of file
+Within a few seconds the Sandbox should be ready for use!
+
+<Image img={require("/img/sandbox.png")} />
+
+## Great, but I want to know more about it
+
+Aztec's Layer 2 network is a fully programmable combined private/public ZK rollup. To achieve this, the network contains the following primary components:
+
+- Archiver - Syncs with L1 and maintains a read-only repository of on-L1 state.
+- World State - The collection of Merkle Trees.
+- P2P - Currently just a locally stored collection of pending transactions.
+- Sequencer - Responsible for ordering transactions into the rollup, executing the rollup circuits and publishing blocks.
+- Aztec Node - Aggregates the above components and provides a facade to 'clients'.
+
+- Aztec RPC Server - Normally residing with the end client, this decrypts and stores a client's private state, executes simulations and submits transactions to the Aztec Node.
+- Aztec.js - Aztec's client library for interacting with the Aztec RPC Server (think Ethers.js).
+
+All of this is included in the Sandbox, with the exception of Aztec.js which you can use to interact with it.
+
+With the help of Aztec.js you will be able to:
+
+- Create an account
+- Deploy a contract
+- Call view methods on contracts
+- Simulate the calling of contract functions
+- Send transactions to the network
+- Be notified when transactions settle
+- Retrieve and view unencrypted logs emitted by contracts
+- Query chain state such as chain id, block number etc.
+
+## I have the Sandbox running, show me how to use it!
+
+Let's have a complete walkthrough from start to finish. I'm using WSL2 Ubuntu under Windows but the following should work under regular Linux or MacOS. We will deploy and use a private token contract on our Sandbox. Writing the contract itself is out of scope for this tutorial, we will use the Private Token Contract supplied as one of the example contracts.
+
+Let's create an empty project called `private-token`. If you are familiar with setting up Typescript projects then you can fast-forward the next couple of steps.
+
+```
+phil@LAPTOP-E72241SF:~$ node -v
+v18.8.0
+phil@LAPTOP-E72241SF:~$ mkdir private-token
+phil@LAPTOP-E72241SF:~$ cd private-token
+phil@LAPTOP-E72241SF:~/private-token$ yarn init
```

Comment:
`yarn init -y` is your friend for this kind of examples

Reply:
Also, should we clarify that we'll be using `yarn` for this example, but devs can also use `npm` if they want to?